src/components/widgets/gcode-preview/GcodePreview.vue

```diff
@@ -276,6 +285,19 @@ export default class GcodePreview extends Mixins(StateMixin) {
     return this.panning ? 'optimizeSpeed' : 'geometricPrecision'
   }
 
+  get showExcludeObjects () {
+    const file = this.$store.getters['gcodePreview/getFile']
+    if (!file) {
+      return true
```

Comment:
I believe this should be `false` (no file loaded, so don't show any exclude objects)
```suggestion
      return false
```
If true, I see the cancel objects even if the current file loaded is not the one printing!

Reply:
Actually, this change doesn't seem right. It's there so you can cancel objects without having to have the gcode preview loaded. I tried this, and it looks like if the gcode preview has another file loaded it's disabled accordingly..

Reply:
Ah I see!

However, it seems a bit weird to me to show the cancellation object on the gcode preview, but no gcode preview...

Reply:
I've tried it and UX wise it's fine I think.. And it has the big advantage of not having to parse the entire Gcode file on the client